doc/contribute/build_ccf.rst

```diff
@@ -33,6 +33,10 @@ Alternatively, on a non-SGX machine, you can build a `virtual` instance of CCF:
 
     CCF defaults to building in the `RelWithDebInfo <https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html>`_ configuration.
 
+.. warning::
+
+    A machine with at least 32Gb of memory is recommended to build CCF with the default compiler.
+
```

Comment:
CCF builds on `WSL` with 15GB RAM (4GB Swap) though? That's the config of my current `WSL`

Reply:
Peak usage I've observed is near 16Gb, so with 15 + 4, you should be fine. Codespaces don't have swap though, and 16Gb is going to be uncomfortably close, particularly as the code changes. I've gone with "recommended", rather than "required".